vizro-core/hatch.toml

```diff
@@ -1,21 +1,15 @@
 [envs.all]
 
 [[envs.all.matrix]]
-python = ["3.8", "3.9", "3.10", "3.11", "3.12"]
+python = ["3.9", "3.10", "3.11", "3.12", "3.13"]
 
 [envs.all.overrides]
-# Kedro is currently not compatible with Python 3.12 and returns exceptions when trying to run the unit tests on
-# Python 3.12. These exceptions turned out to be difficult to ignore: https://github.com/mckinsey/vizro/pull/216
+# Kedro is currently not compatible with Python 3.13 and returns exceptions when trying to run the unit tests on
```

Comment:
Did this directly translate, ie now 3.12 is possible, but 3.13 is not 😂 ?

Reply:
Surprisingly yes! I tested it out and they are following the same scheme as before for Python 3.13.